README.md

```diff
@@ -2,149 +2,140 @@
 [//]: # "To regenerate it, use `npm run generate-scaffolding`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# [Google Cloud Speech API: Node.js Client](https://github.com/googleapis/nodejs-speech)
+# Google Cloud Speech API: Node.js Samples
```

Comment:
@JustinBeckwith this PR replaced the root README with the samples/README.md. Was that intended?

Reply:
🤦‍♂️  No, I don't think that was the intent.  Nice catch @stephenplusplus.  Filed #171 to track.

Reply:
Oh, crud. I should've caught this. Thanks @stephenplusplus!